Add blog post on update coordinator change (#2817)
home-assistant/developers.home-assistant: blog/2025-10-05-coordinator-retrigger.md

---
author: Joakim Plate
authorURL: https://github.com/elupus
title: "Update coordinator now allow retriggering"
---

> The update coordinator with debounce:er active will now accept a request for update

Reviewer: ```suggestion
The update coordinator with debouncer active will now accept a request for update
```

while an update is currently in progress. The request will be queued up to be performed
after the current update finishes.

Consider the following case:

```python
async def _update():
    a = await get_a()
    # A: User or other logic request a new update here through async_schedule_update()
    b = await get_b()
    return (a,b)
```

> A user or code that request an update at timestamp `A` expects that the entities linked will

Reviewer: ```suggestion
A user or code that request an updates at timestamp `A` expects that the entities linked will
```

> all get new data from that time. However, since we previously ignore that request, entities

Reviewer: ```suggestion
all get new data from that time. However, since we previously ignored that request, entities
```

> would have data for the value of `a` that is from a time before update request that was ignored.

Reviewer: ```suggestion
would have data for the value of `a` that is from a time before the update request that was ignored.
```
